example/example-model/src/main/java/uk/gov/gchq/palisade/example/client/ExampleConfigurator.java

```diff
@@ -116,12 +114,41 @@ public static InitialConfigurationService setupSingleJVMConfigurationService() {
         return configService;
     }
 
+    /**
+     * Create resource service for the single JVM example. This method will configure the service as well with example
+     * details.
+     *
+     * @param configService the Palisade configuration service
+     * @param cache         the Palisade cache service
+     * @param palisade      the Palisade service
+     * @return a configured resource service
+     */
+    private static HDFSResourceService createSingleJVMResourceService(final InitialConfigurationService configService, final CacheService cache, final SimplePalisadeService palisade) {
+        HDFSResourceService resource = null;
+        HDFSDataReader reader = null;
+        try {
+            resource = new HDFSResourceService(new Configuration(), cache);
```

Comment:
Can we change this to follow the same builder pattern we have been using else where, e.g new HDFSResourceService().conf(new Configuration).cache(cache);